### PR TITLE
[#169862087] Add info to handle PagoPA compatibility for mobile app

### PR DESCRIPTION
**Pull request by BurnedMarshal**

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please include a description of your PR changes.
  </td>
    </tr>
  </tbody>
</table>



## Affected stories

  * 🌟 [#169862087](https://www.pivotaltracker.com/story/show/169862087): Il backend deve esporre qual è la versione minima dell'app che supporta la corrente implementazione di pagoPA

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: api_public.yaml [hunk 1]
```diff
@@ -33,9 +34,12 @@ definitions:
         type: string
       minAppVersion:
         type: string
+      minAppVersionPagoPa:
```
Comment (gunzip) [inline api_public.yaml]: why aren't these snake_cased as all the other APIs props ?

Comment (BurnedMarshal) [reply api_public.yaml]: Perhaps due to a mistake while introducing the `minAppVersion` property. Now is it better to use the snake format for `minAppVersionPagoPa` and have an API that returns mixed format for properties name? The `minAppVersion` property cannot be changed for compatibility issues.

Comment (BurnedMarshal) [reply api_public.yaml]: I'll check if `minAppVersion` is really used into the production app

Comment (BurnedMarshal) [reply api_public.yaml]: The property `minAppVersion` is not used in the app yet. The PR teamdigitale/io-app#1377 that could use it is not yet merged. I'll do the refactoring.

Comment (BurnedMarshal) [reply api_public.yaml]: Fixed into f632158

**Pull request merged**